cli/lib/tasks/download.js

```diff
@@ -180,6 +190,7 @@ const downloadFromUrl = ({ url, downloadDestination, progress }) => {
 
     const req = request({
       url,
+      proxy: getProxyUrl(),
```

Comment:
proxy is one of those parameters that we should log above, just like we are `debug` logging `url`.